src/sinks/redis.rs

```diff
@@ -143,6 +143,10 @@ impl SinkConfig for RedisSinkConfig {
     fn sink_type(&self) -> &'static str {
         "redis"
     }
+
+    fn can_acknowledge(&self) -> bool {
+        false
```

Comment:
The result of a batch query is known here, so this can probably be true

Reply:
I think so, yes. Fixed in 62b55dfeb